### PR TITLE
Add scrollbar

### PR DESCRIPTION
**Pull request by paulotijero**
Add open user action and it stays that way:
original
Add style scrollbar

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -94,7 +94,7 @@
       <div class="channel-option">
         <svg><use xlink:href="#bel"></svg>
         <svg><use xlink:href="#pin"></svg>
-        <svg><use xlink:href="#people"></svg>
+        <svg id="btnPeople" onclick="showMemberList()" class=""><use xlink:href="#people"></svg>
```
Comment (diegotc86) [inline public/index.html]: This the first `onclick` call we make on the HTML file. Should we use an Event Listener on our JS file instead? What do you think?

File: public/assets/css/style.css [hunk 2]
```diff
@@ -457,6 +461,28 @@ body {
...
+.scrollbar::-webkit-scrollbar {
+  width: 10px;
+  margin-left: 5px;
+}
+
+.scrollbar::-webkit-scrollbar-thumb {
+  border-radius: 50px;
+  background-color: #202225;
+}
+
```
Comment (Linzeur) [inline public/assets/css/style.css]: Thanks so much for the scroll bars, it improves much the visualization and accessibility

File: public/index.html [hunk 3]
```diff
@@ -278,29 +278,10 @@
...
-                    <li class="container-user">
-                      <span class="user">Brian</span>
-                      <span class="date">hoy a las 14:51</span>
-                    </li>
-                    <li class="talk">Asi te queria encontrar!</li>
-                    <li class="talk">Señooooooooorrr!!!</li>
-                    <li class="talk">Señooooooooorrr!!!</li>
-                  </ul>
-                </div>
-              </li> -->
```
Comment (Linzeur) [inline public/index.html]: Good for deleting commented code

File: public/index.html (hunk 1 above)
Comment (Linzeur) [reply public/index.html]: I agree with you, it would be to refactor the code. I think that we could create an issue for that, What do you think?

Comment (diegotc86) [reply public/index.html]: Create an issue please

Comment (Linzeur) [reply public/index.html]: Issue created: #69

**Pull request merged**